compose/cli/command.py

```diff
@@ -75,18 +73,14 @@ def get_client(verbose=False, version=None):
     return client
 
 
-def get_project(base_dir, config_path=None, project_name=None, verbose=False,
-                use_networking=False, network_driver=None):
+def get_project(base_dir, config_path=None, project_name=None, verbose=False):
     config_details = config.find(base_dir, config_path)
+    project_name = get_project_name(config_details.working_dir, project_name)
+    config_data = config.load(config_details)
+    api_version = '1.21' if config_data.version < 2 else None
```

Comment:
Since we're planning on supporting this for a little while longer, maybe we should have a mapping of config_version -> api_version instead of the constant `DEFAULT_API_VERSION`.  

That way the default api versions are defined together in the same place.